src/classes/GhosteryDebug.js

```diff
@@ -0,0 +1,876 @@
+	 */
+	static _helpPromoMessages = [
+		THANKS,
+		'Try our desktop tracker blocker & VPN Midnight for free',
+		'Try our tracker research & analytics extension Insights for free',
+		'Visit ghostery.com to learn more about our values and products',
+	];
+
+	/**
+	 * @private
+	 * @since 8.5.3
+	 *
+	 * Prepares and returns the help strings array for the requested function.
+	 * Exists as a separate function so that public methods can concatenate some custom messages
+	 * with standard help output before forwarding all the string to `_typeset` and `_printToConsole`.
+	 *
+	 * @param {String}	fnName		The name of the function for which help output was requested.
+	 * @return {Array} 				Returns the help strings array for the requested function, or an error strings array if the argument is missing or not supported.
+	 */
+	static _assembleHelpStringArr(fnName) {
+		const {
+			_helpMainMenu,
+			_helpAvailableFunctions,
+			helpFetchABTestsWithIr,
+			helpGetABTests,
+			helpGetConfData,
+			helpGetGlobals,
+			helpShowPromoModal,
+			helpSettingsShow,
+			helpSettingsToggleLogging,
+			helpSettingsToggleOutputStyle,
+		} = GhosteryDebug;
+
+		const invalidArgumentError = [
+			`${CSS_MAINHEADER}'${fnName}' is not a GED function. Here are the valid ones:`,
+			'',
+			..._helpAvailableFunctions,
+		];
+
+		const helpStringArr = [];
+		const eeFnName = (fnName && typeof fnName === 'string' && fnName.toLowerCase()) || undefined;
+		if 		(fnName === undefined) 				helpStringArr.push(..._helpMainMenu);
+		else if (eeFnName === 'getabtests') 		helpStringArr.push(...helpGetABTests);
+		else if (eeFnName === 'getconfdata')		helpStringArr.push(...helpGetConfData);
+		else if (eeFnName === 'getglobals')			helpStringArr.push(...helpGetGlobals);
+		else if (eeFnName === 'fetchabtestswithir')	helpStringArr.push(...helpFetchABTestsWithIr);
+		else if (eeFnName === 'showpromomodal') {
+			helpStringArr.push(...helpShowPromoModal);
+			const activeModalTypes = PromoModals.getActiveModalTypes();
+			activeModalTypes.forEach((amt) => {
+				const { val: cappedAmt } = capitalize(amt.toLowerCase());
+				helpStringArr.push([
+					`'${amt}'`, `Open the ${cappedAmt} modal at the next opportunity`,
+				]);
+			});
+		} else if (eeFnName === 'show')				helpStringArr.push(...helpSettingsShow);
+		else if (eeFnName === 'togglelogging')		helpStringArr.push(...helpSettingsToggleLogging);
+		else if (eeFnName === 'toggleoutputstyle')	helpStringArr.push(...helpSettingsToggleOutputStyle);
+		else 										helpStringArr.push(...invalidArgumentError);
+
+		return helpStringArr;
+	}
+
+	/**
+	 * @since 8.5.3
+	 *
+	 * Prints general and function-specific help output. Part of the public CLI.
+	 *
+	 * @param {String}	[fnName]	The name of the function for which help output was requested, if any.
+	 * @return {String} 			An ad / thank you message (printed to the console as the last line of output).
+	 */
+	// eslint-disable-next-line class-methods-use-this
+	help(fnName) {
```

Comment:
This method doesn't appear on the public `ghostery` object in the console.  Changing the signature to `help = (fnName) => {` shows it.

I think it makes sense to have all available functions/properties appear when typing `ghostery` to the console.